optimum/onnxruntime/modeling_seq2seq.py

```diff
@@ -0,0 +1,623 @@
+            The decoder with past key values model file name overwriting the default file name, allowing to save
+            the decoder model with a different name.
+"""
+
+ENCODER_INPUTS_DOCSTRING = r"""
+    Arguments:
+        input_ids (`torch.LongTensor`):
+            Indices of input sequence tokens in the vocabulary of shape `(batch_size, encoder_sequence_length)`.
+        attention_mask (`torch.LongTensor`):
+            Mask to avoid performing attention on padding token indices, of shape
+            `(batch_size, encoder_sequence_length)`. Mask values selected in `[0, 1]`.
+"""
+
+
+DECODER_INPUTS_DOCSTRING = r"""
+    Arguments:
+        input_ids (`torch.LongTensor`):
+            Indices of decoder input sequence tokens in the vocabulary of shape `(batch_size, decoder_sequence_length)`.
+        encoder_hidden_states (`torch.FloatTensor`):
+            The encoder `last_hidden_state` of shape `(batch_size, encoder_sequence_length, hidden_size)`.
+        encoder_attention_mask (`torch.LongTensor`, *optional*):
+            Mask to avoid performing cross-attention on padding tokens indices of encoder `input_ids`.
+        past_key_values (`tuple(tuple(torch.FloatTensor), *optional*)`
+            Contains the precomputed key and value hidden states of the attention blocks used to speed up decoding.
+            The tuple is of length `config.n_layers` with each tuple having 2 tensors of shape
+            `(batch_size, num_heads, decoder_sequence_length, embed_size_per_head)` and 2 additional tensors of shape
+            `(batch_size, num_heads, encoder_sequence_length, embed_size_per_head)`.
+"""
+
+SEQ2SEQ_ONNX_MODEL_DOCSTRING = r"""
+    Arguments:
+        input_ids (`torch.LongTensor`):
+            Indices of input sequence tokens in the vocabulary of shape `(batch_size, encoder_sequence_length)`.
+        attention_mask (`torch.LongTensor`):
+            Mask to avoid performing attention on padding token indices, of shape
+            `(batch_size, encoder_sequence_length)`. Mask values selected in `[0, 1]`.
+        decoder_input_ids (`torch.LongTensor`):
+            Indices of decoder input sequence tokens in the vocabulary of shape `(batch_size, decoder_sequence_length)`.
+        encoder_outputs (`torch.FloatTensor`):
+            The encoder `last_hidden_state` of shape `(batch_size, encoder_sequence_length, hidden_size)`.
+        past_key_values (`tuple(tuple(torch.FloatTensor), *optional*)`
+            Contains the precomputed key and value hidden states of the attention blocks used to speed up decoding.
+            The tuple is of length `config.n_layers` with each tuple having 2 tensors of shape
+            `(batch_size, num_heads, decoder_sequence_length, embed_size_per_head)` and 2 additional tensors of shape
+            `(batch_size, num_heads, encoder_sequence_length, embed_size_per_head)`.
+"""
+
+_TOKENIZER_FOR_DOC = "AutoTokenizer"
+
+TRANSLATION_SAMPLE = r"""
```

Comment:
Nit: in #226 we changed "SAMPLE" by "EXAMPLE"

Reply:
Good catch ! Thanks @regisss